2_gyum/jisigin.py

```
@@ -0,0 +1,178 @@
+# - requests.get(url): 지정한 URL에 HTTP GET 요청을 보냅니다.
+# - BeautifulSoup(html, 'html.parser'): 응답 받은 HTML을 BeautifulSoup을 사용해 파싱합니다.
+
+# In[1]:
+
+
+from bs4 import BeautifulSoup
+import requests
+
+# 네이버 지식인 삼성전자 검색 페이지 URL
+url = "https://kin.naver.com/search/list.naver?query=%EC%82%BC%EC%84%B1%EC%A0%84%EC%9E%90"
+response = requests.get(url)  # 요청 보내기
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)